# EncoderData.msg

int32 countLeft
float32 countSpeedLeft
int32 countRight
float32 countSpeedRight